posts/2020-09-10-wg-prio-call-for-contributors.md

```diff
@@ -0,0 +1,68 @@
+
+Here is a bit more [comprehensive description][wg-prio]. How we work is detailed [on the Rust Forge](https://forge.rust-lang.org/compiler/prioritization.html).
+
+Our tooling is mainly the [triagebot](https://github.com/rust-lang/triagebot), a trustful messenger that helps us by sending notification to our [Zulip stream][zulip-wg-prio] when an issue on Github is labelled.
+
+We also have a repository with some [issues and meta-issues](https://github.com/rust-lang/compiler-team-prioritization/issues), where we basically note down how we would like our workflow to evolve. Contributions to these issues are welcome, but a bit more context about the workflow of this Working Group is probably necessary.
+
+Documentation is also a fundamental part of the onboarding package that we provide to newcomers. As we basically "organize and sort stuff", a lot happens without writing a single line of code but rather applying procedures to optimize triaging and issues prioritization.
+
+This requires our workflow to be as efficient and well documented as possible. As such, we are always open to contributions to clarify the documentation (and fresh eyeballs are especially precious for that!).
+
+## The typical week of a WG-prioritization member
+
+Our week starts on Thursday/Friday after the Rust Compiler Team meeting (one of the cool teams that keep that beast at bay). We summarize the meeting and draft an agenda for the following one.
+
+In the following days the WG-prioritization and other teams will asynchronously monitor the issue tracker - everyone at their own pace, when time allows - trying to assign a priority to new issues. This greatly helps the various teams (compiler team, standard library team, documentation team and so on) to sort and prioritize their work.
+
+If the issue priority is not immediately clear, it will be tagged with a temporary label and briefly discussed on Zulip by the WG-prioritization: is this issue critical? Is it clear? Does it need a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) (often abbreviated in `MCVE`) or even better a [bisect](https://github.com/rust-lang/cargo-bisect-rustc) to find a regression (we love contributors bisecting code)? We then assign the priority by choosing a value in a range from `P-low` to `P-critical`. The rationale behind the priority levels is detailed in our [guide](https://forge.rust-lang.org/compiler/prioritization/priority-levels.html).
+
+The day before the meeting the agenda is finalized (gathering also the last inputs from other teams) and finally handed to the Compiler Team.
```

Comment:
```suggestion
The day before the meeting we start filling the agenda using triagebot and announce it to the Compiler Team. Then we keep updating the agenda with the latest changes that happen in issues until right before the meeting starts.
```

Reply:
same as above: technically correct but - with an external eye - I would find it confusing

Reply:
Yeah, then I'd remove gathering also the last inputs from other teams. Making another change suggestion ...